lib/mutant/meta.rb

```diff
@@ -14,9 +14,9 @@ class Example
       # Add example
       #
       # @return [undefined]
-      def self.add(&block)
+      def self.add(type = nil, &block)
```

Comment:
These do not intend to have a semantic difference. But actually: Why not change the DSL so the type has to be passed always, this means touching all meta examples, but thats the better way to do. I'll update the PR.